### PR TITLE
handle DICOM files without a valid header

### PR DESCRIPTION
**Pull request by fogleman**
@zachmullen This fixes the Emory issue. Some of their DICOM files don't have a valid header but can still be parsed with a `force=True` option. But this also meant extra steps needed to be taken to avoid reading entire files that aren't actually DICOM. This is done with `defer_size=1024`.

### CODE REVIEW

File: plugins/dicom_viewer/server/__init__.py [hunk 1]
```diff
@@ -65,12 +65,22 @@ def coerce(x):
         return None
 
 
-def process_file(f):
+def parse_file(f):
```
Comment (zachmullen) [inline plugins/dicom_viewer/server/__init__.py]: Any reason to change this name? It's a public symbol, so this is technically a compatibility break.

Comment (fogleman) [inline plugins/dicom_viewer/server/__init__.py]: `process_file` is still there but now calls `parse_file`. I did it to have an early return.

Comment (zachmullen) [inline plugins/dicom_viewer/server/__init__.py]: Whoops, totally missed that.

**Comment by fogleman** [comment]
@zachmullen Why do I see changes unrelated to my PR in the codecov details?

https://codecov.io/gh/girder/girder/compare/e34549d653c63c7fc4a5ae49b5f9b773974e8122...7d5523fa8e747a18cada234e5cdb3a51457936ba/changes

**Comment by zachmullen** [comment]
That may be diffing against master, the diff vis is here: https://codecov.io/gh/girder/girder/compare/e34549d653c63c7fc4a5ae49b5f9b773974e8122...7d5523fa8e747a18cada234e5cdb3a51457936ba/diff

**Pull request merged**